src/utils/Highlight.js

```diff
@@ -0,0 +1,21 @@
+import LazyLoader from './LazyLoader';
+
+export default class Highlight {
+
+    static hljsResolve() {}
+
+    constructor() {
+        if(!Highlight.isReady) {
```

Comment:
Put this here to handle the case in which there are multiple `dcc.Markdown` elements that need syntax highlighting; we won't create new promises for each of them.